WebRoot/js/zimbraMail/share/view/dialog/ZmRevokeShareDialog.js

```diff
@@ -74,7 +74,7 @@ function(share) {
 
 	var params = isPubShare ? ZmMsg.shareWithPublic : isGuestShare ? share.grantee.id : isAllShare ? ZmMsg.shareWithAll :
 					(share.grantee.name || ZmMsg.userUnknown);
-	this._confirmMsgEl.innerHTML = this._formatter.format(params);
+	this._confirmMsgEl.innerHTML = this._formatter.format(AjxStringUtil.htmlEncode(params));
```

Comment:
Why do we need to encode params? Shouldn't we only be encoding the share.grantee.id and/or share.grantee.name instead?